src/Symfony/Component/Form/composer.json

```diff
@@ -33,7 +33,10 @@
         "symfony/http-foundation": ""
     },
     "autoload": {
-        "psr-0": { "Symfony\\Component\\Form\\": "" }
+        "psr-0": { "Symfony\\Component\\Form\\": "" },
+        "exclude-from-classmap": [
+            "/Tests/"
```

Comment:
`autoload-dev` only affects the root project. So this would only work when you are running tests for Symfony itself.